adcc/adc_pp/matrix.py

```diff
@@ -411,9 +411,9 @@ def adc3_i1(hf, mp, intermediates):
         p0_ov = intermediates.cvs_p0_ov
         p0_vv = intermediates.cvs_p0_vv
     else:
-        p0_ov = mp.mp2_diffdm[b.ov]
-        p0_oo = mp.mp2_diffdm[b.oo]
-        p0_vv = mp.mp2_diffdm[b.vv]
+        p0_ov = mp.mp2_diffdm.ov
+        p0_oo = mp.mp2_diffdm.oo
+        p0_vv = mp.mp2_diffdm.vv
```

Comment:
Just `p0 = mp.mp2_diffdm`?

Reply:
Okay, but then I need to modify the CVS intermediate stuff above, too 👍